src/main/java/karuna/karuna_backend/security/config/WebSecurityConfig.java

```diff
@@ -42,9 +42,10 @@ SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
                                 "/swagger-ui/**",
                                 "/swagger-ui.html",
                                 "/v3/api-docs/**",
-                                "/swagger-resources/**",
-                                "/contents/**")
+                                "/swagger-resources/**")
                         .permitAll()
+                        .requestMatchers("/contents/**")
```

Comment:
One more thing, it should not be contents/** anyway. It should be contenst/specific-path since I believe we will use that for getters as well for normal users.

Reply:
fixed